mlflow/gateway/config.py

```diff
@@ -106,7 +110,21 @@ def validate_field_compatibility(cls, config: Dict[str, Any]):
         else:
             raise MlflowException.invalid_parameter_value(f"Invalid OpenAI API type '{api_type}'")
 
-        return config
+        return info
+
+    if IS_PYDANTIC_V2:
+        from pydantic import model_validator as _model_validator
```

Comment:
Sphinx thinks `model_validator` is a class member and attempts to document it without an alias.